Servers/utils/userStatusCodes.utils.ts

```
@@ -0,0 +1,12 @@
+export class STATUS_CODES {
+    static 201 = (data: any) => { return { data } }
+    static 200 = (data: any) => { return { data } }
+    static 400 = (data: any) => { return { data } }
+    static 404 = (k: string, v: any) => { return { data: `user with ${k}: ${v} not found` } };
+    static 500 = (error: any) => {
+        return {
+            data: "internal server error",
+            errorDetails: error.toString()
+        }
+    };
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)